runtime/src/bank.rs

```diff
@@ -20151,4 +20174,86 @@ pub(crate) mod tests {
         bank.apply_feature_activations(ApplyFeatureActivationsCaller::NewFromParent, false);
         assert_eq!(bank.hashes_per_tick, Some(DEFAULT_HASHES_PER_TICK));
     }
+
+    #[test]
+    fn test_stake_account_consistency_with_rent_epoch_max_feature() {
+        // this test can be removed once set_exempt_rent_epoch_max gets activated
+        solana_logger::setup();
+        for rent_epoch_max_enabled_initially in [false, true] {
```

Comment:
you can use `test_case` instead of these `for` loops:
https://crates.io/crates/test-case

for example:
https://github.com/solana-labs/solana/blob/180ea1e0a/ledger/src/shred/merkle.rs#L1241-L1244

Reply:
done